app.rb

```diff
@@ -122,30 +122,72 @@ def list_rentals
     end
   end
 
-  def save_data
-    save_data_to_json('books.json', @books)
-    save_data_to_json('people.json', @person)
-    save_data_to_json('rentals.json', @rentals)
+  def get_data(file_name)
+    if File.exist?("#{file_name}.json")
+      File.read("#{file_name}.json")
+    else
+      empty_json = [].to_json
+      File.write("#{file_name}.json", empty_json)
+      empty_json
+    end
   end
```

Comment:
- [ ] Dear @yegonkimutai, as suggested by the previous reviewer and as stated clearly in the project requirement, On exit from your app we need to make sure the data is being saved (all the data: books, people, and rentals) into JSON files. In your case after exiting the app, no data is being saved to the JSON files, and no data is retrieved on the app start. Please make sure data are always saved to the JSON files and retrieved on the app start.

### This is my experience
![image](https://github.com/yegonkimutai/OOP-school-library/assets/69077061/c6b9a70a-ac9f-4632-b2ee-a645f6bc991c)

Reply:
well resolved! ✔️